app/models/presenters/submission_plate_downstream_completed_presenter.rb

```diff
@@ -0,0 +1,245 @@
+        transition pending: :passed
+      end
+
+      state :pending do
+        include Statemachine::StateDoesNotAllowChildCreation
+      end
+
+      state :started do
+        include Statemachine::StateDoesNotAllowChildCreation
+      end
+
+      state :processed_1 do
+        include Statemachine::StateDoesNotAllowChildCreation
+      end
+
+      state :processed_2 do
+        include Statemachine::StateDoesNotAllowChildCreation
+      end
+
+      state :processed_3 do
+        include Statemachine::StateDoesNotAllowChildCreation
+      end
+
+      state :processed_4 do
+        include Statemachine::StateDoesNotAllowChildCreation
+      end
+
+      state :passed do
+        include Statemachine::StateAllowsChildCreation
+
+        # We only show the submission options sidebar if we are allowed to create a new submission
+        def sidebar_partial
+          return 'submission_default' if allow_new_submission?
+
+          'default'
+        end
+      end
+
+      state :qc_complete, human_name: 'QC Complete' do
+        include Statemachine::StateAllowsChildCreation
+      end
+
+      state :cancelled do
+        include Statemachine::StateDoesNotAllowChildCreation
+      end
+
+      state :failed do
+        include Statemachine::StateDoesNotAllowChildCreation
+      end
+
+      state :unknown do
+        include Statemachine::StateDoesNotAllowChildCreation
+      end
+    end
+
+    include Presenters::Statemachine::AllowsLibraryPassing
+    include Presenters::SubmissionBehaviour
+
+    self.summary_items = {
+      'Barcode' => :barcode,
+      'Number of wells' => :number_of_wells,
+      'Plate type' => :purpose_name,
+      'Current plate state' => :state,
+      'Input plate barcode' => :input_barcode,
+      'Created on' => :created_on
+    }
+
+    DESCENDANT_TUBE_INCLUDES =
+      'receptacle,aliquots,aliquots.request,aliquots.request.request_type,receptacle.requests_as_source.request_type'
+
+    # Overridden from SubmissionBehaviour
+    # Check for submission already in progress
+    # If not, check that the first sequencing run has completed by checking the downstream tubes
+    def allow_new_submission?
+      # No more than one submission of a type can be active at time for a given labware.
+      # Prevent new submissions if any are currently in progress, as the submission type
+      # is currently not available.
+      submissions_in_progress = pending_submissions? || active_submissions?
+      return false if submissions_in_progress
+
+      # Next check if any downstream tubes exist matching the requirements (there may be more than one
+      # depending on pooling and repeats, we need at least one to have completed the first run)
+      downstream_sequenced_tubes = find_downstream_sequenced_tubes
+
+      # If there are no downstream tube(s) yet, we have completed the first run so we cannot create the
+      # new submission
+      return false if downstream_sequenced_tubes.blank?
```

Comment:
This can be merged with line 123 to be `return downstream_sequenced_tubes.present?` so returns true if exist and false if they dont.